src/components/Dropdowns/SelectionDropdown.tsx

```diff
@@ -26,16 +37,23 @@ type SelectionDropdownProps = {
   showTooltip?: boolean;
   /** Width of the dropdown. Overrides the default sizing behavior if set. */
   width?: string | null;
-  style?: React.CSSProperties;
+  /**
+   * Whether the search bar should be enabled. If enabled, will show search bar and filter
+   * by search input when the total number of items is above `searchThresholdCount`. True by default.
+   */
+  enableSearch?: boolean;
+  /** The number of items that must be in the original list before the search bar will be shown. 10 by default.*/
+  searchThresholdCount?: number;
```

Comment:
Nit: it seems like these two could be just one single thing. like if enableSearchThresholdCount === -1 (default value?) then force disable search.  idk, maybe it's better with two separate

Reply:
I like keeping them separate so disabling the searchbar feels a little more intuitive:

```
<SelectionDropdown enableSearch={false} />
vs.
<SelectionDropdown searchThresholdCount={-1}>
```